### PR TITLE
Volto requires Yarn 3 now. Update make start-frontend output

### PR DESCRIPTION
**Pull request by fredvd**
As discussed in this morning Volto Team meeting.

### CODE REVIEW

File: docs/install/install-from-packages.md [hunk 1]
```diff
@@ -145,20 +145,21 @@ npm install -g yo
...
-    npm install yarn@1
+    npm install yarn@3
     ```
 
-2.  Verify that Yarn v1.x.x is installed and activated.
+2.  Verify that Yarn v3.x.x is installed and activated.
 
     ```shell
     yarn -v
+    3.2.3
```
Comment (stevepiercy) [inline docs/install/install-from-packages.md:164]: Commands go in `shell`, output to `console`.

````suggestion
    ```
    ```console
    3.2.3
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One small syntax change, and good to merge!

**Comment by stevepiercy** [comment]
Please check other files:

- glossary.md has a link to Classic Yarn
- install-from-packages

And in the volto submodule:

- volto/addons/index.md line 335
- volto/getting-started/install.md (yikes! this is out of date.)

File: docs/install/install-from-packages.md (hunk 1 above)
Comment (fredvd) [reply docs/install/install-from-packages.md:164]: In the other console outputs, both commands and outputs are copy/pasted in one block, for example the whole cookiecutter flow. That's the example I followed.

Comment (stevepiercy) [reply docs/install/install-from-packages.md:164]: Which ones? If syntaxes are mixed in a block, then they need to be broken apart.

**Comment by fredvd** [comment]
Fixes applied, ready to merge.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Thank you! As soon as it's green, it can be merged.

**Comment by stevepiercy** [comment]
New issue created for the volto submodule bits:
https://github.com/plone/volto/issues/4441

The broken test is from linkcheckbroken and MyST releasing 0.19 earlier today. I'll merge this PR and create a new issue to handle the broken links.

**Comment by stevepiercy** [comment]
I have an open PR to fix the build: https://github.com/plone/documentation/pull/1459 Once that is green and passes, I'll merge that, to avoid a broken build here.

**Comment by fredvd** [comment]
thank you. Yes that is what I was worried about, that it might break the whole build if the docs check now fails.  The single time I push something directly to main.   :-$

**Pull request merged**